R/main.R

```diff
@@ -1,10 +1,28 @@
-#' shiny.i18n: A package for Shiny applications internationalization.
+#' i18n
 #'
-#' The shiny.i18n package provides functions for easy Shiny applications internationalization.#'
+#' Returns a translator object which can be used for translations.
 #'
-#' @section Foo functions:
-#' The foo functions ...
 #'
-#' @docType package
-#' @name shiny.i18n
-NULL
+#' @param translation_csvs_path character with path to folder with CSV files
+#' @param translation_json_path character with path to JSON translation file.
+#' The format of the JSON you can find on shiny.i18n documentation website.
+#' @param translation_csv_config yaml file with config fields described (used
+#' only when in CSV files mode.).
+#'
+#' @return Translator object
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' n18i <- i18n(translation_csvs_path = "path/to/folder/with/csv/translations")
+#' n18i <- i18n(translation_json_path = "path/to/json/file")
+#' n18i$set_translation_language("pl")
+#' n18i$t("Some text")
+#' }
+i18n <- function(translation_csvs_path = NULL,
```

Comment:
what is the added value of this function? I don't see any...
why don't we simply remove it and document how you should create a translator by callying `Translator$new`?

every line of code comes with a cost, it's best to avoid writing code if we can.

Reply:
Agree.